api/lib/application/sco-organization-learners/sco-organization-learner-controller.js

```diff
@@ -180,6 +182,33 @@ const checkScoAccountRecovery = async function (
   );
 };
 
+const updateOrganizationLearnersPassword = async function (request, h) {
+  const payload = request.payload.data.attributes;
+  const userId = request.auth.credentials.userId;
+  const organizationId = payload['organization-id'];
+  const organizationLearnersId = payload['organization-learners-id'];
+
+  const generatedCsvContent = await DomainTransaction.execute(async (domainTransaction) => {
+    const organizationLearnersGeneratedPassword = await usecases.updateOrganizationLearnersPassword({
+      userId,
+      organizationId,
+      organizationLearnersId,
+      domainTransaction,
+    });
+    return usecases.generateResetOrganizationLearnersPassword({
+      organizationLearnersGeneratedPassword,
+    });
```

Comment:
Renommer ce usecase pour indiquer clairement qu'il génère un fichier CSV : 

```suggestion
    return usecases.generateResetOrganizationLearnersPasswordCsvContent({
      organizationLearnersGeneratedPassword,
    });
```